src/tensorlake/cli/secrets.py

```diff
@@ -134,7 +133,7 @@ def unset(auth: AuthContext, secret_names: str):
 
 def _get_all_existing_secrets(auth: AuthContext) -> List[dict]:
     resp = auth.client.get(
-        f"/platform/v1/organizations/{auth.organization_id}/projects/{auth.project_id}/secrets?limit=100"
+        f"/platform/v1/organizations/{auth.organization_id}/projects/{auth.project_id}/secrets?pageSize=100"
```

Comment:
do we expect a user to have fewer than 100 secrets?

Reply:
more than 100 is very unlikely, if it gets to a high number we'll need to rethink the CLI UX.